src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -70,6 +71,9 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
   def withRetry(work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit): Unit = {
     work onComplete {
       case Success(s) => onSuccess(s)
+      case Failure(e: GoogleJsonResponseException) if e.getStatusCode == 403 =>
+        logger.error(e.getMessage, e)
+        throw e
```

Comment:
This doesn't change the 10 retries... basically there's a retry within a retry here, this catches it *after* it failed the 10 retries.